### PR TITLE
Adding Support for Python3.10

### PR DESCRIPTION
**Pull request by abukhoy**
Here to support python3.10, created a python3.10 enviroment and changed the torch version to 2.0.0 only.

Tested on both local machines x86 and ARM.

### CODE REVIEW

**Comment by quic-rishinr** [comment]
@abukhoy It seems the line endings have changed, causing all lines to appear as modified. Could you please switch the line endings to either ‘CRLF’ or ‘LF’ in your code editor and push the changes again?

**Comment by quic-amitraj** [comment]
LGTM

**Review by quic-amitraj — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ RUN mkdir -p /app/qefficient-library
 COPY . /app/qefficient-library
 
 # Create Virtual Env for the docker image
```
Comment (quic-rishinr) [inline Dockerfile:26]: Shouldn't we add the step for installing python3-venv in the readme?

Comment (abukhoy) [reply Dockerfile:26]: Added.

**Comment by asmigosw** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -59,9 +59,9 @@ For other models, there is comprehensive documentation to inspire upon the chang
 ## Quick Installation
 ```bash
 
-# Create Python virtual env and activate it. (Recommended Python 3.8)
-
-python3.8 -m venv qeff_env
+# Create Python virtual env and activate it. (Recommended Python 3.10)
+sudo apt install python3.10-venv
```
Comment (quic-amitraj) [inline README.md:63]: There is no need of this line 63, remove it.

**Review by quic-amitraj — CHANGES_REQUESTED** [review]

**Review by quic-amitraj — APPROVED** [review]

**Pull request merged**